index.js

```diff
@@ -44,13 +44,16 @@ const audits = [
   require('./audits/manifest/start-url')
 ];
 
-Gatherer
-    .gather(gatherers, {url, driver})
-    .then(artifacts => Auditor.audit(artifacts, audits))
-    .then(results => {
-      console.log(results);
-    }).catch(function(err) {
-      console.log('error encountered', err);
-      console.log(err.stack);
-      throw err;
-    });
+module.exports = function(opts) {
+  var url = opts.url || defaultUrl;
+  Gatherer
+      .gather(gatherers, {url, driver})
+      .then(artifacts => Auditor.audit(artifacts, audits))
+      .then(results => {
+        console.log(results);
```

Comment:
console.log is a direct proxy to stdout with a newline inserted at the end :goat: